nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -1328,9 +1328,9 @@
                 var tooltip = nf.Common.formatPropertyTooltip(propertyDescriptor, propertyHistory);
 
                 if (nf.Common.isDefinedAndNotNull(tooltip)) {
-                    infoIcon.qtip($.extend({
-                        content: tooltip,
-                    }, nf.Common.config.tooltipConfig));
+                    infoIcon.qtip($.extend(nf.Common.config.tooltipConfig, {
```

Comment:
@scottyaslan - If you checkout the description of $.extend [1] it shows the first argument is the target meaning that is what object that is being modified/updated. I don't we want to be modifying the tooltip configuration template like we are here as it will affect all subsequent $.extend calls. 

This applies to all of modified $.extend lines in this PR.

[1] https://api.jquery.com/jquery.extend/